packages/create-sitecore-jss/src/templates/nextjs-sxa/src/pages/_app.tsx

```diff
@@ -0,0 +1,20 @@
+import type { AppProps } from 'next/app';
+import { I18nProvider } from 'next-localization';
+
+import 'assets/style.css';
+import 'assets/app.css';
```

Comment:
can we add all styles at the bottom of the `app.css`? In this case you don't need to change `_app.tsx`